### ISSUE TITLE
PET loads training data when just "do_eval" is set

### ISSUE THREAD

**Issue by puraminy** [timoschick/pet#40]
I trained a model using pet and I want to evaluate it over some validation set. While I remove `--do_train` and only leave `--do_eval` option, it still loads training data which in my opinion is unnecessary,
